store/postgres/src/filter.rs

```diff
@@ -317,8 +317,8 @@ fn store_filter_by_mode<'a>(
 
         StoreFilter::EndsWith(..) | StoreFilter::NotEndsWith(..) => {
             let (attribute, op, value) = match filter {
-                StoreFilter::StartsWith(attribute, value) => (attribute, " LIKE ", value),
-                StoreFilter::NotStartsWith(attribute, value) => (attribute, " NOT LIKE ", value),
+                StoreFilter::EndsWith(attribute, value) => (attribute, " LIKE ", value),
+                StoreFilter::NotEndsWith(attribute, value) => (attribute, " NOT LIKE ", value),
```

Comment:
Good catch, these new filters don't have unit tests, what do you think of taking the opportunity to add a test for an`EndsWith` query in `postgres/test/store.rs`?